pyccel/ast/datatypes.py

```diff
@@ -77,23 +78,24 @@
 )
 
 #==============================================================================
-default_precision = {'real': 8, 'int': 8, 'complex': 8, 'bool':4, 'float':8}
-dtype_and_precision_registry = { 'real':('real',8),
-                                 'double':('real',8),
-                                 'float':('real',8),       # sympy.Float
-                                 'pythonfloat':('real',8), # built-in float
+
+default_precision = {'real': 8, 'int': numpy.dtype(int).alignment, 'complex': 8, 'bool':4, 'float':8}
```

Comment:
When is `real` used? I wonder if we could remove this key from all dictionaries in this module.

Reply:
it is used everywhere a Variable can be declared only by `real` not `float` or `double`

Reply:
So `real` is used for the precision of Pyccel variables? And `float` represents a Python float? What about `double` then, where is it used?

Reply:
everything is converted to `real`